resources/wikia/modules/uuid.js

```diff
@@ -0,0 +1,8 @@
+define('uuid', function () {
```

Comment:
This comes from [here](https://stackoverflow.com/questions/105034/create-guid-uuid-in-javascript/2117523#2117523), right? Maybe we could include that link in the method docs for more context?

Reply:
actually we copied it from here: https://github.com/Wikia/app/blob/62beff8152ae38d1377cc1c187d5aa8dcc61946b/extensions/wikia/Track/templates/tracking_cookies.mustache#L5, but including link to StackOverflow is a good point

Reply:
done